config/boards/radxa-zero-3w.csc

```diff
@@ -0,0 +1,50 @@
+# Rockchip RK3566 quad core
+BOARD_NAME="Radxa ZERO 3W"
+BOARDFAMILY="rk35xx"
+BOARD_MAINTAINER=""
+BOOTCONFIG="radxa-zero3-rk3566_defconfig"
+KERNEL_TARGET="vendor,current,edge"
+KERNEL_TEST_TARGET="vendor"
+FULL_DESKTOP="yes"
+BOOT_LOGO="desktop"
+BOOT_FDT_FILE="rockchip/rk3566-radxa-zero-3w.dtb"
+IMAGE_PARTITION_TABLE="gpt"
+BOOT_SCENARIO="spl-blobs"
+BOOTFS_TYPE="fat" # Only for vendor/legacy
+AIC8800_TYPE="sdio"
```

Comment:
I don't understand why this split is necessary. Is it just about the additional wifi extension? If so I guess a simple `if` can handle whether this should come into play or not.

Reply:
The DTB files are split in the latest linux-dtb-* packages -- the old one doesn't even exist anymore.

Reply:
Hm. First I was like 'let's just add some `if BOARD = 3w, then install wifi stuff` but then realized this isn't going to work, because board model is detected at actual boot.
So either split models or install wifi drivers on both models where on 3E they are simply never used. I guess former is better.